AutoFlow_OmicsDataHandling/thermo/io.py

```diff
@@ -0,0 +1,134 @@
+"""I/O.
+
+This module hosts functions to load cobra models, create and adjust tFBA-ready
+models.
+"""
+
+import os.path
+
+from cobra.io import load_json_model
+from pytfa import ThermoModel
+from pytfa.io import (
+    load_thermoDB,
+    read_lexicon,
+    read_compartment_data,
+    annotate_from_lexicon,
+    apply_compartment_data,
+)
+
+from AutoFlow_OmicsDataHandling.thermo import static
+
+
+def _path(filename):
+    # Returns the path to a file in the static folder
+    return os.path.join(os.path.dirname(static.__file__), filename)
+
+
+def load_cbm(model_name):
```

Comment:
Is this function shared by any of the other modules we have so far (e.g., FVA, sampling, or the modules that Svetlana wrote for TMFA?).  I am just thinking if it makes sense to have a common io module for the repository.

Reply:
I would agree that it is a good idea to have a common IO module (or at least shared between modules that build upon cobrapy, such as FVA).

Reply:
Can you make a new issue for this in Jira?